ironbank/build_ironbank_docker.sh

```diff
@@ -97,20 +97,23 @@ function build_dockerfile {
   OUTPUT_PATH=ironbank/$(basename $1)
   REQUIREMENTS="$OUTPUT_PATH/docker_packages_metadata.txt"
 
-  if [[ ! -f $REQUIREMENTS ]] && [[ ! -f $1/Dockerfile.ironbank ]]; then
-    echo "docker_packages_metadata.txt is missing in this docker, please create Dockerfile.ironbank, aborting..."
-    return 1;
-  fi
-
 
   if [[ ! -d $OUTPUT_PATH ]]; then
     mkdir $OUTPUT_PATH
   fi
   if [[ -f $1/Dockerfile.ironbank ]]; then
     # if we have a special Dockerfile for ironbank, copy it instead of generating
     cp $1/Dockerfile.ironbank $OUTPUT_PATH/Dockerfile
+  # if requirements.txt exists execute build_dockerfile with requirements_file_exists=truw
+  elif [[ -f $REQUIREMENTS ]]; then
+    python ./ironbank/build_dockerfile.py --docker_image_dir $1 --output_path $OUTPUT_PATH --requirements_file_exists true
   else
-    python ./ironbank/build_dockerfile.py --docker_image_dir $1 --output_path $OUTPUT_PATH
+    python ./ironbank/build_dockerfile.py --docker_image_dir $1 --output_path $OUTPUT_PATH --requirements_file_exists false
+  fi
+
+  if [[ ! -f $REQUIREMENTS ]] && [[ ! -f $1/Dockerfile.ironbank ]]; then
+    echo "docker_packages_metadata.txt is missing in this docker, please create Dockerfile.ironbank, aborting..."
+    return 1;
```

Comment:
No need for this condition, is not it?

Reply:
I keep this condition just in case we will change the flow in the future, so we will have one more validation